sdk/keyvault/azkeys/client.go

```diff
@@ -1109,12 +1154,12 @@ func (c *Client) RestoreKeyBackup(ctx context.Context, keyBackup []byte, options
 // ImportKeyOptions contains the optional parameters for the Client.ImportKeyOptions parameter
 type ImportKeyOptions struct {
 	// Whether to import as a hardware key (HSM) or software key.
-	Hsm *bool `json:"Hsm,omitempty"`
+	HardwareProtected *bool `json:"Hsm,omitempty"`
```

Comment:
Should this be
```suggestion
	HardwareProtected *bool `json:"hsm,omitempty"`
```

Reply:
No, in the [swagger](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.3/keys.json#L1727) it is capitalized